src/main/java/coverage2/Home.java

```diff
@@ -13,4 +14,12 @@ public Home(String name,String address1){
     public String getName(){
         return name;
     }
+
+    public String getAddress2() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube2.tools.kmb.sonynei.net/coding_rules#rule_key=squid%3AUndocumentedApi)